From 7c7b66bcd7d8d08467f4b0a9e4debe5d8804a5d3 Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@streamunlimited.com>
Date: Wed, 4 May 2016 15:31:09 +0200
Subject: [PATCH] net: wireless: mwifiex: Disable 11ac connections in sta mode

There is no other way how to disable 11ac. Use this hack. Suggested by:
http://article.gmane.org/gmane.linux.kernel.wireless.general/151776

Signed-off-by: Marek Belisko <marek.belisko@streamunlimited.com>
---
 drivers/net/wireless/mwifiex/cmdevt.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/wireless/mwifiex/cmdevt.c b/drivers/net/wireless/mwifiex/cmdevt.c
index c5a14ff..58e6efd 100644
--- a/drivers/net/wireless/mwifiex/cmdevt.c
+++ b/drivers/net/wireless/mwifiex/cmdevt.c
@@ -1467,6 +1467,8 @@ int mwifiex_ret_get_hw_spec(struct mwifiex_private *priv,
 	else
 		adapter->fw_bands = BAND_B;
 
+	adapter->fw_bands &= ~BAND_AAC;
+
 	adapter->config_bands = adapter->fw_bands;
 
 	if (adapter->fw_bands & BAND_A) {
-- 
1.9.1

